src/popup/pages/CreatePassword/index.js

```diff
@@ -92,28 +92,31 @@ class CreatePassword extends React.Component {
           return v;
         })
       }, () => {
-        this.setBtnStatus()
+        this.checkConfirmStatus()
       })
     })
   }
+  checkConfirmStatus=()=>{
+    if (this.state.confirmPwd.length > 0 && this.state.inputPwd !== this.state.confirmPwd) {
+      this.callSetState({
+        errorTip: getLanguage('passwordDifferent')
```

Comment:
This is fix for https://github.com/oasisprotocol/oasis-wallet-ext/issues/52

Reply:
yes , thank for discuss, i will make new PR about this